nucypher/utilities/prometheus/metrics.py

```diff
@@ -21,19 +20,13 @@
 
 class PrometheusMetricsConfig:
     """Prometheus configuration."""
-    def __init__(self,
-                 port: int,
-                 listen_address: str = '',  # default to localhost ip
-                 collection_interval: int = 90,  # every 1.5 minutes
-                 start_now: bool = False):
-
-        if not port:
-            raise ValueError('port must be provided')
 
-        self.port = port
-        self.listen_address = listen_address
-        self.collection_interval = collection_interval
-        self.start_now = start_now
+    def __init__(self):
+        self.port = 9101
```

Comment:
Forcing 9101 is tricky because other services may be run on the same machine, including other things that use prometheus. In fact, we probably have this situation on lynx-2 at the moment. It has a Porter that uses prometheus and probably uses port 9101 for its metrics.